client/my-sites/backup/rewind-flow/granular-restore.tsx

```diff
@@ -556,9 +568,10 @@ const BackupGranularRestoreFlow: FunctionComponent< Props > = ( {
 		}
 
 		if ( isFinished ) {
+			dispatch( recordTracksEvent( 'calypso_jetpack_backup_granular_restore_completed' ) );
```

Comment:
On my tests, At the end of a first restore, the event is triggered two times. After that, if without reloading the site, I go back to a backup and select again a file to restore, the event is triggered before starting the restore

Reply:
@mavegaf Thanks for the feedback!

> At the end of a first restore, the event is triggered two times.

This is now fixed!

> After that, if without reloading the site, I go back to a backup and select again a file to restore, the event is triggered before starting the restore

This is because if you try to initiate a new restore, it still has the `finished` status of the previous one with the same `rewindId`, so it shows the restore success view for a bit, until the API fetches the new restore.

The issue is not necessarily related to this PR, and I take a look and it might require some sort of refactoring, so I think it would be best to tackle it in a separate PR. Would that work for you?

Could you validate if the first issue reported is resolved and if the Tracks event is triggering properly?